### ISSUE TITLE
Error: Cannot find module '../build/Release/sharp.node' on Travis-CI, using yarn, with caching

### ISSUE THREAD

**Issue by vinz243** [lovell/sharp#747]
Hi, 

I'm trying to use sharp in my travis-ci builds. Here is the `.travis.yml`

```yml
language: node_js

env:
  - CXX=g++-4.8
addons:
  apt:
    sources:
      - ubuntu-toolchain-r-test
    packages:
      - g++-4.8

cache:
  yarn: false
  directories:
    - node_modules

dist: trusty
sudo: required

node_js:
  - "7"

script:
  - yarn test
  # - yarn backend:coverage-travis -- -v

cache:
  directories:
    - node_modules

install:
  - yarn
  - cp ./secure.json.template ./secure.json
  - sudo add-apt-repository ppa:mc3man/trusty-media -y
  - sudo apt-get update -q
  - sudo apt-get install ffmpeg -y
  - ffmpeg -version
  - ffprobe -version
```

You can find the repo here:

https://github.com/vinz243/cassette/tree/features/new-model

And the log https://travis-ci.org/vinz243/cassette/jobs/216372111

**Comment by lovell** [comment]
Hello, are you able to try with caching disabled, or with npm instead of yarn, to remove those as possible causes?

**Comment by vinz243** [comment]
Thanks! Caching was raising this issue. Any way to enable it with sharp to speed up build tho?

**Comment by lovell** [comment]
Travis CI appears to be using yarn v0.17.8 but the latest available is v0.22.0. There have been a number of recent improvements to its caching behaviour so perhaps try updating if that's possible.
